contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/check_gpio.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+import argparse
+import os
+from contextlib import contextmanager
+
```

Comment:
```suggestion
# This file is part of Checkbox.
#
# Copyright 2024 Canonical Ltd.
# Written by:
#   Rick Wu <rick.wu@canonical.com>
#
# Checkbox is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License version 3,
# as published by the Free Software Foundation.
#
# Checkbox is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with Checkbox. If not, see <http://www.gnu.org/licenses/>
```